service/src/main/kotlin/fi/espoo/evaka/children/ChildControllerCitizen.kt

```diff
@@ -78,21 +83,61 @@ class ChildControllerCitizen(private val accessControl: AccessControl) {
         val defaultServiceNeedOptions =
             tx.getServiceNeedOptions()
                 .filter { it.defaultOption }
-                .associateBy({ it.validPlacementType }, { ServiceNeedOptionPublicInfo.of(it) })
+                .associateBy { it.validPlacementType }
         val serviceNeedDateRanges = serviceNeeds.map { FiniteDateRange(it.startDate, it.endDate) }
         return tx.getPlacementSummary(childId).flatMap { placement ->
             val placementRange = FiniteDateRange(placement.startDate, placement.endDate)
             placementRange.complement(serviceNeedDateRanges).map {
+                val defaultServiceNeedOption = defaultServiceNeedOptions[placement.type]
                 ServiceNeedSummary(
                     it.start,
                     it.end,
-                    defaultServiceNeedOptions[placement.type],
+                    defaultServiceNeedOption?.let { sno -> ServiceNeedOptionPublicInfo.of(sno) },
+                    defaultServiceNeedOption?.contractDaysPerMonth,
                     placement.unit.name
                 )
             }
         }
     }
 
+    @GetMapping("/{childId}/attendance-summary/{yearMonth}")
+    fun getChildAttendanceSummary(
+        db: Database,
+        user: AuthenticatedUser.Citizen,
+        clock: EvakaClock,
+        @PathVariable childId: ChildId,
+        @PathVariable yearMonth: YearMonth
+    ): AttendanceSummary {
+        return db.connect { dbc ->
+                dbc.read { tx ->
+                    accessControl.requirePermissionFor(
+                        tx,
+                        user,
+                        clock,
+                        Action.Citizen.Child.READ_ATTENDANCE_SUMMARY,
+                        childId
+                    )
+                    val range = FiniteDateRange(yearMonth.atDay(1), yearMonth.atEndOfMonth())
+                    val reservationDays = tx.countReservationDays(childId, range)
+                    val attendanceDays = tx.countChildAttendanceDays(childId, range)
+                    val absenceDays =
```

Comment:
is this actually `unplannedAbsenceDays` or something since it's limited to a set of types?